src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -35,23 +35,29 @@ import '../header_logo.scss';
 import { OpenSearchDashboardsLogoDarkMode } from './opensearch_dashboards_logo_darkmode';
 
 /**
- * @param {string} logoUrl - custom URL for the top left logo of the main screen
+ * @param {string} fullLogoUrl - custom URL for the top left logo of the main screen
+ * @param {string} logoUrl - custom URL for the logo icon
+ * @param {string} title - custom title for the application
  */
 export interface CustomLogoType {
+  fullLogoUrl?: string;
   logoUrl?: string;
+  title: string;
 }
 
 export const CustomLogo = ({ ...branding }: CustomLogoType) => {
-  return !branding.logoUrl ? (
-    OpenSearchDashboardsLogoDarkMode()
-  ) : (
-    <img
-      data-test-subj="customLogo"
-      data-test-image-url={branding.logoUrl}
-      src={branding.logoUrl}
-      alt="logo"
-      loading="lazy"
-      className="logoImage"
-    />
-  );
+  if (!branding.fullLogoUrl && !branding.logoUrl) {
```

Comment:
NIT: this can still be handled by a ternary operator. i'm fine for a readability if you think it's easier to turn into to an if statement if it's going to be if this then return else return then probably dont need the else.

for example:
```
if this
    return this
else that
    return that
```
    
can be transformed to
```
if this
    return this
return that
```

unless you plan on doing logic in the else

Reply:
Turns out that ternary operator for this logic looks good too, changed to ternary operator.